tests/sentry/ingest/test_transaction_clusterer.py

```diff
@@ -200,7 +202,7 @@ def test_run_clusterer_task(cluster_projects_delay, default_organization):
         project2 = Project(id=223, name="project2", organization_id=default_organization.id)
         for project in (project1, project2):
             project.save()
-            for i in range(len(project.name)):
+            for i in range(10):
```

Comment:
The reason I made this change is to calculate easier the `MAX_SET_SIZE` above, and the same for the `5` below (which is just the merge threshold limit).